cmd/ethrex_replay/src/cli.rs

```diff
@@ -0,0 +1,118 @@
+use clap::{Parser, Subcommand};
+
+use crate::bench::run_and_measure;
+use crate::fetcher::{get_blockdata, or_latest};
+use crate::run::{exec, prove};
+
+pub const VERSION_STRING: &str = env!("CARGO_PKG_VERSION");
+
+#[derive(Parser)]
+#[command(name="Ethrex_replay_cli", author, version=VERSION_STRING, about, long_about = None)]
```

Comment:
I think it's better to use the binary name as command name (`env!("CARGO_BINARY_NAME")`). That's make the tool more consistent.